tests/index.test.js

```diff
@@ -0,0 +1,95 @@
+const request = require('supertest');
+const app = require('../index');
+const nock = require('nock');
+const mockSearchData = require('../mocks/searchData.json');
+const mockStylistData = require('../mocks/stylistData.json');
+const mockReviewData = require('../mocks/reviewData.json');
+const mockErrorData = require('../mocks/errorData.json');
+const mockErrorLocationData = require('../mocks/errorLocationData.json');
+const mockErrorValidationData = require('../mocks/errorValidationData.json');
+
+describe('GET results', () => {
+  it('should get search results based on a term and location', async () => {
+    const term = 'interlocks';
+    const location = 'boston';
+    nock('https://api.yelp.com')
+      .get(
```

Comment:
I do wonder about this kind of testing with mock data - it's essentially testing that if I put this specific mock data into whatever I'm testing, I should get it out...so I'm not sure that it's valuable long-term.